extension/src/notification/Widget.tsx

```diff
@@ -0,0 +1,100 @@
+import CloseIcon from "@mui/icons-material/Close";
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
+import {
+  Card,
+  CardHeader,
+  CardContent,
+  CardActions,
+  IconButton,
+  Checkbox,
+  FormControlLabel,
+} from "@mui/material";
+import { FC, memo } from "react";
+
+import { WidgetContext } from "../shared/context/WidgetContext";
+import { ViewMarkdownViewer } from "../shared/ui-components/common";
+import { PLATEAUVIEW_NOTIFICATION_DOM_ID } from "../shared/ui-components/common/ViewClickAwayListener";
+
+import { useNotificationLogic } from "./hooks/useNotificationLogic";
+
+type NotificationProps = {
+  isEnable?: boolean;
+  content?: string;
+  startTime?: string;
+  finishTime?: string;
+};
+
+type NotificationWidgetProps<NotificationProps> = {
+  inEditor: boolean;
+  widget: {
+    property: {
+      default: NotificationProps;
+    };
+  };
+};
+
+type Props = NotificationWidgetProps<NotificationProps>;
+
+export const Widget: FC<Props> = memo(function WidgetPresenter({ widget }) {
+  const { ready, visible, show, content, handleClose, handleCheckboxChange, doNotShowAgain } =
+    useNotificationLogic();
+
+  if (!visible) return null;
+
+  return (
+    <div id={PLATEAUVIEW_NOTIFICATION_DOM_ID}>
+      <WidgetContext
+        isEnable={widget.property.default.isEnable}
+        content={widget.property.default.content}
+        startTime={widget.property.default.startTime}
+        finishTime={widget.property.default.finishTime}>
+        {ready && show && (
+          <Card
+            sx={{
+              width: "349px",
+              borderRadius: "6px",
+            }}>
```

Comment:
```suggestion
            sx={{
              position: "absolute",
              right: 0,
              bottom: 0,
              width: isMobile ? "calc(100vw - 16px)" : "349px",
              borderRadius: "6px",
            }}>
```
It's a little bit tricky here.
We're using widget align system (WAS) to locate the widgets, but actually that can't meed our requirement. For most widgets of View 3.0 we didn't actually follow the 'correct' way that WAS expected.
In order to avlid Notification widget effect Inspector etc. we can position it as `absolute` and calc the width manually on mobile layout.
Why `-16`: we should set 8px padding in all directions for all widget areas (Search / Inspector / Notification)

Reply:
I see! It looks good, thank you telling me👍